Source Main 5.2/source/w_ObjectInfo.h

```diff
@@ -141,7 +141,7 @@ class OBJECT
     BYTE          Kind;
     WORD		Skill;
     BYTE		  m_byNumCloth;
-    BYTE		  m_byHurtByOneToOne;
+    BYTE		  m_byHurtByDeathstab;
```

Comment:
float?

Reply:
i haven't thought of that. let me try that

Reply:
This is the part where it's set,  
`to->m_byHurtByDeathstab = 35;` 
Does this mean it's meant to stay for 35 frames? but now it should stay for the corresponding animation factor?

Reply:
I applied the suggestion, animation and duration seems fine.